event-sourcing-redis/Readme.md

```diff
@@ -0,0 +1,141 @@
+# Event Sourcing with Redis
+It is a common practice to use asynchronous messaging technologies to implement communication between multiple microservices.
+It helps to have scalable, fault tolerant and decoupled architecture. Kafka is go-to solution when it comes to streaming pipelines and publish/subscribe systems.
+You can easily scale microservice architecture by running multiple instances of your microservice in same consumer group, easily add new consumers without modifying existing communication-related code,
+take advantage of persistence of Kafka topics by re-consuming them any time you need and have many more advantages over traditional http communication protocol. 
```

Comment:
Change this sentence to bullet points. I.E. 
Main advantages over traditional HTTP communication protocol:
- You can easily scale microservice architecture by running multiple instances of your microservice in same consumer group.
- Easily add new consumers without modifying existing communication-related code.
- Take advantage of persistence of Kafka topics by re-consuming them any time you need